README.md

```diff
@@ -237,7 +237,9 @@ See the [Springroll Application Docs](https://github.com/SpringRoll/SpringRoll/t
 
 
 ### UISizePlugin, LayersPlugin:
```

Comment:
Should we also add `PointerSizePlugin` and `ButtonSizePlugin` to this list? Is `UISizePlugin` still a thing?